.travis.yml

```diff
@@ -84,6 +99,11 @@ jobs:
     env:
       ANSIBLE_VERSION: devel
 
+  - python: "3.5"
+    <<: *if-cron-or-manual-run-or-tagged
+    env:
+      ANSIBLE_VERSION: devel
+
   - python: "2.7"
     <<: *if-cron-or-manual-run-or-tagged
```

Comment:
@webknjaz should we remove these mixtures so each PR raise/update has all environment combinations tested? It looks like all were tested by travis prior to 4.0.0

Reply:
This was done to avoid overloading usage of Travis CI which has limits and queues not only per repo but also per org. That's why I've set such if-clauses. Keep in mind that we also have Molecule which already consumes a tremendous amount of its resources despite a lot more limitations which I've applied there.